webpack.config.js

```diff
@@ -84,17 +85,53 @@ module.exports = (env) => ({
               icon: true
             }
           },
+          {
+            include: [path.resolve(__dirname, 'src/baseIcons')],
+            loader: '@svgr/webpack',
+            options: {
+              svgo: true,
+              svgoConfig: {
+                plugins: [
+                  { removeAttrs: false },
```

Comment:
Just noticed that removeAttrs should be disabled by default: https://nicedoc.io/svg/svgo#user-content-built-in-plugins. If so, it should not be necessary to set it to false - needs testing though. And should not make any difference to leave as is.

Reply:
As this is used through webpack svgr, might be safest to leave these as is for now.

Reply:
At the moment, it seems that we are still using older version of the svgo than the **v2.4.0** where the `preset-default` was introduced (at least when checked from the yarn.lock).
**yarn.lock**:
```
svgo@^1.2.2:
  version "1.3.2"
```